toolchain/lower/handle.cpp

```diff
@@ -166,14 +167,75 @@ auto HandleBuiltin(FunctionContext& /*context*/, SemIR::InstId /*inst_id*/,
   CARBON_FATAL() << "TODO: Add support: " << inst;
 }
 
+static auto GetCalleeBuiltinFunctionKind(const SemIR::File& sem_ir,
```

Comment:
Is this and HandleBuiltinCall worth a function comment, since it's not a boilerplate handler?

Reply:
Sure.